helm/kanister-operator/values.yaml

```diff
@@ -22,6 +22,7 @@ controller:
   # false : CRDs would be created by helm
   # true : CRDs would be created by kanister controller
   updateCRDs: true
+  parallelism:
```

Comment:
We should have a default value here. zero value of the type would also work, that's what is being used right now I think.

Reply:
For now we're having a default value of 8. I was more leaned towards keeping it none in the helm chart because, in code we're determining the value as if it's set in environmental variable than that will be the value, else the default that is 8

Reply:
if we ever decide to add a `values.schema.json` we will have to add a value here, that was the reason I was recommending to add a value. Even if we add 8 here that would be set in env var can be changes as well right? I don't see something that you can not do, that you are doing now, after adding default value here.

Reply:
Yeah we can add default here !

Reply:
Added Default value as 8